kornia/augmentation/functional.py

```diff
@@ -105,16 +105,16 @@ def random_affine(input: torch.Tensor,
     r"""Random affine transformation of the image keeping center invariant
 
     See :func:`~kornia.augmentation.param_gen._random_affine_gen` for details.
-    See :func:`~kornia.augmentation.functional.apply_affine` for details.
+    See :func:`~kornia.augmentation.functional._apply_affine` for details.
```

Comment:
We should not put link to private functions in docstring, I think instead we can just put a link to the class in here.